lib/vscode/.eslintrc.json

```diff
@@ -799,17 +799,40 @@
 				]
 			},
 			{
-				"target": "**/vs/server/**",
+				"target": "**/vs/server/node/**",
```

Comment:
:fire: really great to have these rules fixed.

Reply:
Heads up - I restored the rules to the defaults, and moved a bunch of our files to other places inside vscode.
Will update the docs on changes. :eyes: